charts/karavi-observability/values.yaml

```diff
@@ -37,6 +37,20 @@ karaviMetricsPowerflex:
     type: ClusterIP
   logLevel: INFO
   logFormat: text
+  authorization:
```

Comment:
Can these new fields for authorization and cert-manager be added to the existing documentation (https://dell.github.io/csm-docs/docs/observability/deployment/helm/).

Reply:
I pushed changes to https://github.com/dell/csm-docs/pull/121. The cert-manager documentation should be at the root level of Observability: https://dell.github.io/csm-docs/docs/observability/. Unless there is something I missed?